### PR TITLE
handle property missing in GET first

### LINKED ISSUE #162: GET missing property bug
If you try to GET a property that is missing from a widget, the code crashes:

Error in console:
```
BicycleIndividual:16 testButton: {"func":"GET","property":"clickCount"}
VM28:1 Uncaught (in promise) SyntaxError: Unexpected token u in JSON at position 0
    at JSON.parse (<anonymous>)
    at Button.click (BicycleIndividual:16)
    at inputHandler (BicycleIndividual:16)
    at BicycleIndividual:16
```

Test:
[GET_missing_property.zip](https://github.com/ArnoldSmith86/virtualtabletop/files/5881975/GET_missing_property.zip)

### PR DESCRIPTION
**Pull request by robartsd**
fixes #162 

Test case loaded to room. To reset the test case, choose "GET missing property bug" from the "Development" game in the games list.

Clicking button updates label to "Success!" after adding/incrementing a "clickCount" property.

### CODE REVIEW

**Comment by ArnoldSmith86** [comment]
PR-SERVER-BOT: You can play around with it here: http://212.47.248.129:3163/pr-test (or any other room on that server)

File: client/js/widgets/button.js [hunk 1]
```diff
@@ -260,8 +260,13 @@ export class Button extends Widget {
           switch(a.aggregation) {
           case 'first':
             if(collections[a.collection].length)
```
Comment (ArnoldSmith86) [inline client/js/widgets/button.js:262]: I usually add curly brackets to the whole construct as soon as either the `if` or the `else` branch contains more than one instruction.

I have no problem merging it as is though.

File: client/js/widgets/button.js [hunk 2]
```diff
@@ -260,8 +260,13 @@ export class Button extends Widget {
...
-              // always get a deep copy and not object references
-              variables[a.variable] = JSON.parse(JSON.stringify(collections[a.collection][0].p(a.property)));
+              if(collections[a.collection][0].p(a.property) !== undefined)
+                // always get a deep copy and not object references
+                variables[a.variable] = JSON.parse(JSON.stringify(collections[a.collection][0].p(a.property)));
+              else {
+                variables[a.variable] = null;
+                problems.push(`Property ${a.property} missing from first item of collection, setting ${a.variable} to null`);
```
Comment (ArnoldSmith86) [inline client/js/widgets/button.js:268]: I think all other problem messages have a period at the end.

**Review by ArnoldSmith86 — APPROVED** [review]
Code looks fine. Just added some nitpicking. :D

**Comment by robartsd** [comment]
I've made the style changes suggested by @ArnoldSmith86 .

**Review by ArnoldSmith86 — APPROVED** [review]
The outer `if` still has more than one line without braces but you don't _have_ to change that.

**Pull request merged**